apps/ca.c

```diff
@@ -33,6 +33,7 @@
 
 #include "apps.h"
 #include "progs.h"
+#include <ca_logic.h>
```

Comment:
C trick :  a #include <header.h>  is about including a system (or compiler) header first if it exist, them after any project file which filename matches.

So this project header file should be included like the others : 
`#include "ca_logic.h" `

Reply:
Yes, of course. I had other issues and tried that as a fix (didn't help) and forgot to change it back... Will do that right away, but won't push it.